util/envDynawo.sh

```diff
@@ -209,6 +209,31 @@ export_var_env() {
   export $name="$value"
 }
 
+export_preload() {
```

Comment:
tcmalloc is less performant in mono threading than the system malloc, so why do we add it in dynawo whereas there is no multi-threading? Wouldn't it be more coherent to add this as a 3rd party in dynawo-algorithm and use it there?